examples/fscloud/version.tf

```diff
@@ -4,7 +4,7 @@ terraform {
     # Pin to the lowest provider version of the range defined in the main module's version.tf to ensure lowest version still works
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "1.56.1"
+      version = "1.62.0"
```

Comment:
the minimum version in the module should also be updated, to prevent using a version that is lower than 1.62.0
https://github.com/terraform-ibm-modules/terraform-ibm-cbr/blob/d2d6e8fd379d35b88cbd427d00e234dcb84018cc/modules/fscloud/version.tf#L6